lib/semian/net_http.rb

```diff
@@ -87,6 +87,16 @@ def disabled?
       raw_semian_options.nil?
     end
 
+    def acquire_semian_resource(*)
+      result = super
+
+      if result.is_a?(::Net::HTTPServerError) && raw_semian_options.fetch(:open_circuit_server_errors)
```

Comment:
Wait... `Net::HTTP` returns a constant as the result on a request on `5xx`? That seems highly strange :s